crates/wit-component/dl/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "dl"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = [ "staticlib" ]
+
+[dependencies]
+
+[workspace]
```

Comment:
Mind throwing this in the same toplevel workspace to avoid an extra `Cargo.lock` file?